pyproject.toml

```diff
@@ -27,18 +27,19 @@ markers = [
 
 [tool.ruff]
 # Enable Pyflakes `E` and `F` codes by default.
-select = [
+lint.select = [
     "C40",
     "C416",
     "C419",
     "E",
     "F",
     "PIE810",
     "SIM101",
+    "SLOT"
 ]
 
 # Ignore rules that currently fail on the SymPy codebase
-ignore = [
+lint.ignore = [
```

Comment:
Easier way to update this is probably to create a [ruff.tool.lint] section. This works too of course.

Reply:
Also is ruff on the latest version on Sympy's CI? This is a recent change the latest SemVar update to ruff 0.2

Reply:
The github tests use ruff 0.2.3 , I couldn't find ruff version specified anywhere in the repository , the workflow file just runs `pip install ruff` withour specifying a version so I assume the latest version is being used .

Reply:
> Easier way to update this is probably to create a [ruff.tool.lint] section. This works too of course.

I did not want to change the convention that was going on earlier , I will change it if it's necessary .